scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -969,6 +970,19 @@ def processPipelineJsonForExistingGates():
         raise e
 
 
+def activateSpinnakerSession(cookie):
+    try:
+        empty_json = json.loads('{}')         
+        api_url = sapor_host_url+"/oes/appOnboarding/spinnaker/pipeline/stage"        
+        headers = {'Content-Type': 'application/json', 'x-user-cookie' : cookie,
+                   'x-spinnaker-user': isd_admin_username}
+        response = requests.post(url=api_url, headers=headers, data=json.dumps(empty_json))     
```

Comment:
response status code here is expected to be 200 or 201 (success) or 401 (is session expired). If anything other thatn these, why don't we throw an exception and break the script execution so that remaining part doesn't get executed ?